src/main/java/ir/sahab/uncaughtexceptionrule/UncaughtException.java

```diff
@@ -0,0 +1,30 @@
+package ir.sahab.uncaughtexceptionrule;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Exception that contains several exceptions that are occurred concurrently
```

Comment:
Wrapper around all exceptions that are occurred during unit test execution.

@see UncaughtExceptionBase# afterHandle()